aldryn_people/models.py

```diff
@@ -111,7 +117,11 @@ class Meta:
         verbose_name_plural = _('People')
 
     def __str__(self):
-        return self.name
+        pkstr = str(self.pk)
+
+        if six.PY2:
+            pkstr = six.u(pkstr)
+        return self.name.strip() or pkstr
```

Comment:
No, that's missing the point. What I'm looking for is some help understanding what the value returning from strip() might be. Without it, I have to run the program in my mind, and guess. If what you're looking for is the empty string (did I guess correctly?) then you could help me by writing it like this:
```
name = self.name.strip()
if len(name) > 0:
    return name
else:
    return pkstr
```
or
```
name = self.name.strip()
return name if len(name) > 0 else pkstr
```